schematic/manifest/examples/generator_usage.py

```diff
@@ -4,13 +4,10 @@
 from schematic.manifest.generator import ManifestGenerator
 
 from schematic.utils.google_api_utils import download_creds_file
-from schematic.utils.config_utils import load_yaml
 
-from definitions import CONFIG_PATH, DATA_PATH
+from schematic import CONFIG
 
-config_data = load_yaml(CONFIG_PATH)
-
-PATH_TO_JSONLD = os.path.join(DATA_PATH, config_data["model"]["input"]["location"])
+PATH_TO_JSONLD = os.path.join(CONFIG.DATA_PATH, CONFIG["model"]["input"]["location"])
```

Comment:
Most of the edits across the module files look like this. 

1. The config file is no longer loading in modules individually. Now, it will get loaded once as soon as one of the attributes is accessed. 
2. Any of the raw variables are prefixed with `CONFIG.`. 
3. Anytime the config dictionary is used, the `CONFIG` object is indexed directly. This is just a shortcut for `CONFIG.DATA[key]`.